docs/content/documentation/getting-started/configuration.md

```diff
@@ -73,6 +73,11 @@ feed_filename = "atom.xml"
 # files are always copied, regardless of this setting.
 hard_link_static = false
 
+# When set to "true", all links are generated with trailing slashes.
+# This is the default due to how most default web server configurations work.
+# DO NOT set this to false if you don't know what you're doing.
```

Comment:
Unless "DO NOT ... if you don't know what you're doing." is a convention in Zola (I didn't check), perhaps it is better to inform what the consequence is. Since the consequence is both severe and relatable and can be explained briefly, how about:

```suggestion
# Trailing slashes change what relative paths (images, links) are relative to.
# Changing this may break all images and relative paths on a site, so be careful.
```